sklearn/manifold/tests/test_spectral_embedding.py

```diff
@@ -181,6 +181,32 @@ def test_spectral_embedding_amg_solver(seed=36):
     assert _check_with_col_sign_flipping(embed_amg, embed_arpack, 0.05)
 
 
+def test_spectral_embedding_amg_solver_failure(seed=36):
+    # Test spectral embedding with amg solver failure
+    try:
```

Comment:
It is cleaner to just use pytest.importorskip here

Reply:
Done